infra/core/host/appservice.bicep

```diff
@@ -33,6 +33,13 @@ param numberOfWorkers int = -1
 param scmDoBuildDuringDeployment bool = false
 param use32BitWorkerProcess bool = false
 
+// Target DB properties
+param connectionStringKey string = 'AZURE-SQL-CONNECTION-STRING'
+param targetResourceId string = ''
+param appUser string = ''
+@secure()
+param appUserPassword string = ''
+
```

Comment:
Why do we need appUser for app service?

Reply:
when setting up connection between the app service and target db, user need to provide target db's connection info including appUser, appPassword.